### DISCUSSION TITLE
Issue with communicating with the server

### DISCUSSION THREAD

**Discussion by everettperiman** [PrismarineJS/mineflayer#2501] [Q&A]
Host: Ubuntu 20.04
Clients: Ubuntu 20.04, Windows 10

I was able to get some of the example scripts to work recently but I am not having issues. When hosting a MC server no matter if I run the client scripts on the host computer or a remote windows pc it keeps giving me the same error pasted below. Any help would be appreciated with this!

Error: unsupported/unknown protocol version: 758, update minecraft-data
    at /home/server/node_modules/minecraft-protocol/src/client/autoVersion.js:32:28
    at /home/server/node_modules/minecraft-protocol/src/ping.js:13:7
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
TypeError: Cannot read property 'minecraftVersion' of undefined
    at /home/server/node_modules/minecraft-protocol/src/client/autoVersion.js:34:42
    at /home/server/node_modules/minecraft-protocol/src/ping.js:13:7
    at processTicksAndRejections (internal/process/task_queues.js:95:5)

**Comment by amoraschi** [comment]
![IMG_20220301_071244.jpg](https://user-images.githubusercontent.com/68395930/156115038-2fe335c1-b17f-40d8-8372-a66469520a52.jpg)

**Reply by Drizzt9000** [reply]
How the hell do I get into a darn server? Please help me.

**Reply by amoraschi** [reply]
Look at the examples, tutorial, and api before asking here

**Reply by Drizzt9000** [reply]
That doesn't help because I don't know what a api is. And I don't know how to code very well.

**Reply by amoraschi** [reply]
Sorry, looks like I didn't link the Discord server, it's in the readme, I think we can help you there
